src/org/labkey/test/tests/elisa/ElisaMultiPlateAssayTest.java

```diff
@@ -34,15 +33,26 @@
 import static org.hamcrest.CoreMatchers.hasItems;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.not;
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
```

Comment:
What's with all these added imports? Are they left over from some experimenting you did?

Reply:
I have IntelliJ configured to "Optimize Imports" when I commit to get rid of unused imports. It also does other things, like, it expanded "`import static org.junit.Assert.*`" into the four methods that are actually used by that test.